drivers/filesystems/fastfat_new/fatdata.h

```diff
@@ -263,11 +263,19 @@ extern LONG FatDebugTraceIndent;
 }
 #endif
 
+#ifndef __REACTOS__
+#define DebugUnwind(X) {                                                      \
+    if (AbnormalTermination()) {                                              \
```

Comment:
Then, following the trend chosen in this PR, why not having instead some sort of
```c
#define AbnormalTermination() _SEH2_AbnormalTermination()
```
somewhere?

Reply:
> initial import commit

I had a look: it will be worth a separate PR.

> `#define AbnormalTermination()`

I don't know how to do that for all SEH2 parts, especially `_SEH2_END`,
so better leave those parts "forked".
I reverted my related change here.